build_projects/dotnet-cli-build/NuGetUtil.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.IO;
```

Comment:
That's a good point. Moving the shared project to target that will probably work, but if we don't do it we'll need to duplicate this between the host and cli builds, which we'll need to do anyways so It's fine to leave it I think.